gammapy/maps/regionnd.py

```diff
@@ -154,6 +155,37 @@ def plot_region(self, ax=None, **kwargs):
         ax = self.geom.plot_region(ax, **kwargs)
         return ax
 
+    def plot_mask(self, ax=None, color="k", label=""):
```

Comment:
I would propose to use the `**kwargs` pattern here, like we do for the other plot functions. Internally we can set default values for the `kwargs` like:

```python3

def plot_something(ax, **kwargs):
    """
    Parameters
    -----------
    **kwargs : dict
        Keyword arguments passed to `~matplotlib.pyplot.axvspan` 
    """
    kwargs.setdefault("color", "k")
    kwargs.setdefault("alpha", 0.05)
    kwargs.setdefault("label", "mask")
    ax.axvspan(xmin, xmax, **kwargs)


```

Reply:
Done.